### PR TITLE
Feature/823 administration bulk upload page

### PR DESCRIPTION
**Pull request by abhishek-p-s**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204271669000633

### CODE REVIEW

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+                    onClick={downloadTemplate}
+                    shape="round"
+                  >
+                    {text.download}
+                  </Button>
+                </Space>
+                <Space align="center" size={32}>
+                  <img src="/assets/data-upload.svg" />
+                  <p>{text.uploadDataLabel}</p>
+                  <Select
```
Comment (dedenbangkit) [inline frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:301]: We don't need this

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+                    onChange={handleChange}
+                    className="custom-select"
+                  >
+                    {forms.map((f, fI) => (
+                      <Option key={fI} value={f.id}>
+                        {f.name}
+                      </Option>
+                    ))}
+                  </Select>
+                  <AdministrationDropdown />
```
Comment (dedenbangkit) [inline frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:313]: We also don't need this

File: frontend/src/App.js [hunk 3]
```diff
@@ -121,6 +122,12 @@ const RouteList = () => {
           path="master-data"
           element={<Private element={MasterData} alias="master-data" />}
         />
+        <Route
+          path="data/upload-administration-data"
```
Comment (dedenbangkit) [inline frontend/src/App.js:126]: `master-data`

File: frontend/src/components/filters/AdministrationFilters.js [hunk 4]
```diff
@@ -49,7 +49,7 @@ const AdministrationFilters = ({
       {["Super Admin"].includes(authUser?.role?.value) && (
         <Col>
           <Space>
-            <Link to="/control-center/data/upload">
+            <Link to="/control-center/data/upload-administration-data">
```
Comment (dedenbangkit) [inline frontend/src/components/filters/AdministrationFilters.js:52]: `master-data`

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+            type: "error",
+            message: text.templateFetchFail,
+          });
+          setLoading(false);
+        });
+    }
+  };
+
+  return (
+    <div id="uploadData">
```
Comment (dedenbangkit) [inline frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:212]: uploadMasterData

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx [hunk 6]
```diff
@@ -0,0 +1,345 @@
...
+    return uiText[activeLang];
+  }, [activeLang]);
+
+  const pagePath = [
+    {
+      title: text.controlCenter,
+      link: "/control-center",
+    },
+    {
+      title: "Administration Data Upload",
```
Comment (dedenbangkit) [inline frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:55]: text.AdministrationDataUpload

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx [hunk 7]
```diff
@@ -0,0 +1,345 @@
...
+    }
+  };
+
+  const downloadTemplate = () => {
+    setLoading(true);
+    if (updateExisting) {
+      exportGenerate();
+    } else {
+      api
+        .get(`export/form/${formId}`, { responseType: "blob" })
```
Comment (dedenbangkit) [inline frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:179]: Use below endpoint
Screenshot_2023-12-29T16_45_35 121018967

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx (hunk 1 above)
Comment (abhishek-p-s) [reply frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:301]: image and text ?

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx (hunk 2 above)
Comment (abhishek-p-s) [reply frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:313]: <AdministrationDropdown /> only this or the entire <Space>....</Space>?

File: frontend/src/App.js (hunk 3 above)
Comment (abhishek-p-s) [reply frontend/src/App.js:126]: instead of the "/upload-administration-data"  we need to replace to master-data right?

Comment (dedenbangkit) [reply frontend/src/App.js:126]: no, `master-data/upload-administration-data`

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx (hunk 1 above)
Comment (dedenbangkit) [reply frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:301]: remove dropdowns, keep image and text

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx (hunk 2 above)
Comment (dedenbangkit) [reply frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:313]: Please keep in mind that this upload, doesn't have any relations with forms. Please re-check all of them

File: frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx (hunk 1 above)
Comment (dedenbangkit) [reply frontend/src/pages/upload-administration-data/UploadAdministrationData.jsx:301]: and change text.uploadDataLabel to text.uploadMasterDataLabel. Create uploadMasterDataLabel in the ui-text

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**